### PR TITLE
enhance: inform user what to do when the version property is missing

### PR DESCRIPTION
**Pull request by colindean**
While setting up a new pipeline, I couldn't quite figure out what I was missing. Turns out, I'd missed something quite important when copypasting from an existing configuration, and neither the server nor the vela CLI could tell me what was wrong! This disambiguates that error message by providing a remedy.

### CODE REVIEW

**Comment by colindean** [comment]
N.b. I did this in GH editor, so I'm bracing for some style errors at minimum ;-)

File: compiler/native/validate.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 func (c *client) Validate(p *yaml.Build) error {
 	// check a version is provided
 	if len(p.Version) == 0 {
-		return fmt.Errorf("no version provided")
+		return fmt.Errorf("no version provided. Add 'version: \"1\"' to the configuration.")
```
[the remark this answers is not in the document]
Comment (colindean) [reply compiler/native/validate.go:17]: That's a very fair assessment. I'll make the change!

[the remark this answers is not in the document]
Comment (colindean) [reply compiler/native/validate.go:17]:
```suggestion
		return fmt.Errorf("no \"version:\" YAML property provided")
```

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Review by wass3r — APPROVED** [review]

**Pull request merged**